pip/pep425tags.py

```diff
@@ -189,6 +236,11 @@ def get_supported(versions=None, noarch=False):
             else:
                 # arch pattern didn't match (?!)
                 arches = [arch]
+        elif sys.platform == 'linux':
+            if is_manylinux1_compatible():
+                arches = [arch, arch.replace('linux', 'manylinux1')]
```

Comment:
@rmcgibbo @njsmith is the order important here? For instance in case both the `linux_x86_64` and the `manylinux1_x86_64` wheel files are available in a wheel index, would pip choose the first or the later?